<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleBar</class>
 <widget class="QWidget" name="TitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TitleBar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
	color: white;
	border-image: transparent;
}
QToolButton
{
	background-color: transparent;
	border-style: flat;
	border-width: 1px;
	border-radius: 3px;
	border-color: rgb(51, 153, 255);
}
QToolButton:hover
{
	background-color: rgba(115, 209, 255, 200);
}
QToolButton:pressed
{
	background-color: rgba(20, 59, 255, 100);
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="500,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SJTU Flow</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonMin">
     <property name="toolTip">
      <string>Minimize</string>
     </property>
     <property name="whatsThis">
      <string>Minimize</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonClose">
     <property name="toolTip">
      <string>Close</string>
     </property>
     <property name="whatsThis">
      <string>Close</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
